x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -135,19 +138,27 @@ class ReportListingUi extends Component<Props, State> {
   public render() {
     return (
       <EuiPageContent horizontalPosition="center" className="euiPageBody--restrictWidth-default">
-        <EuiTitle>
-          <h1>
-            <FormattedMessage id="xpack.reporting.listing.reportstitle" defaultMessage="Reports" />
-          </h1>
-        </EuiTitle>
-        <EuiText color="subdued" size="s">
-          <p>
-            <FormattedMessage
-              id="xpack.reporting.listing.reports.subtitle"
-              defaultMessage="Find reports generated in Kibana applications here"
-            />
-          </p>
-        </EuiText>
+        <EuiFlexGroup justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiTitle>
+              <h1>
+                <FormattedMessage
+                  id="xpack.reporting.listing.reportstitle"
+                  defaultMessage="Reports"
+                />
+              </h1>
+            </EuiTitle>
+            <EuiText color="subdued" size="s">
+              <p>
+                <FormattedMessage
+                  id="xpack.reporting.listing.reports.subtitle"
+                  defaultMessage="Find reports generated in Kibana applications here"
+                />
+              </p>
+            </EuiText>
+            <ReportHelper apiClient={this.props.apiClient} />
+          </EuiFlexItem>
+        </EuiFlexGroup>
```

Comment:
Do we have a design for the placement of this link? I envisioned it would appear below the table. When there are warnings or failures detected in the data, we would also show a box with this link at the top of the table.